docs/features/embed-pages-to-teams.md

```diff
@@ -35,6 +35,7 @@ You can embed any modern SharePoint page in Microsoft Teams by creating a Teams
     - Search initiated from Teams will return search results from Teams, not search results from the embedded site.
     - You cannot create content (news, pages, lists, or libraries) in Teams. You must create content from the SharePoint site using a web browser or SharePoint mobile app.
     - Page comment feature is not available from the SharePoint pages, when they are exposed in Microsoft Teams
+    - The Stream web part for SharePoint doesn't work in teams
```

Comment:
Can you please change this to refer to "Microsoft Stream" and "Microsoft Teams" instead of how it's referenced to be inline with MS Docs rules?